prisma/seed.ts

```diff
@@ -34,6 +34,21 @@ async function main() {
     },
   });
 
+  await prisma.user.upsert({
```

Comment:
Why do we need a new user here? Above we have `"user-1"` already.

Reply:
When a user logs in, the page they see depends on whether they have any projects or not. Users without projects are directed to the welcome screen to create a new project, while users with existing projects are directed to the projects screen. As a result, the first test would fail if we use `"user-1"` since they already have projects.

Reply:
I see, would it be possible to move the creation of that user to the test file? I think we are then good to go with this.

Reply:
The user creation has been moved to the test file @maxdeichmann, thanks for the review.

Reply:
Hi @maxdeichmann , thanks for the feedback. Requested changes implemented.